### PR TITLE
#3383 change readtimeout

### PR DESCRIPTION
**Pull request by agiridharan**
set DefaultReadTimeout value to be 1 min

### CODE REVIEW

**Comment by agiridharan** [comment]
#build

File: dev/fattest.simplicity/src/componenttest/topology/utils/LDAPUtils.java [hunk 1]
```diff
@@ -1,4 +1,5 @@
 /*******************************************************************************
+
```
Comment (jvanhill) [inline dev/fattest.simplicity/src/componenttest/topology/utils/LDAPUtils.java]: Why was this modified?

File: dev/com.ibm.ws.security.wim.core_fat/publish/servers/com.ibm.ws.security.wim.core.fat.noreg/server.xml [hunk 2]
```diff
@@ -12,7 +12,7 @@
 		ldapType="BadLdapType"
 		searchTimeout="8m">
 		<failoverServers name="failoverLdapServers">
-			<server host="${ldap.server.4.name}" port="${ldap.server.4.port}"/>
+			<server host="${ldap.server.5.name}" port="${ldap.server.5.port}"/>
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.core_fat/publish/servers/com.ibm.ws.security.wim.core.fat.noreg/server.xml]: This belongs in PR #5429.

File: dev/com.ibm.ws.security.wim.core_fat/publish/servers/com.ibm.ws.security.wim.core.fat.maxSearchResult/server.xml [hunk 3]
```diff
@@ -12,7 +12,7 @@
 		ldapType="IBM Tivoli Directory Server"
 		searchTimeout="8m">
 		<failoverServers name="failoverLdapServers">
-			<server host="${ldap.server.4.name}" port="${ldap.server.4.port}"/>
+			<server host="${ldap.server.5.name}" port="${ldap.server.5.port}"/>
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.core_fat/publish/servers/com.ibm.ws.security.wim.core.fat.maxSearchResult/server.xml]: This belongs in PR #5429.

File: dev/com.ibm.ws.security.wim.core_fat/publish/servers/com.ibm.ws.security.wim.core.fat.invalidBaseEntryInRealm/server.xml [hunk 4]
```diff
@@ -12,7 +12,7 @@
 		ldapType="IBM Tivoli Directory Server"
 		searchTimeout="8m">
 		<failoverServers name="failoverLdapServers">
-			<server host="${ldap.server.4.name}" port="${ldap.server.4.port}"/>
+			<server host="${ldap.server.5.name}" port="${ldap.server.5.port}"/>
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.core_fat/publish/servers/com.ibm.ws.security.wim.core.fat.invalidBaseEntryInRealm/server.xml]: This belongs in PR #5429.

File: dev/com.ibm.ws.security.wim.core_fat/publish/servers/com.ibm.ws.security.wim.core.fat.dynamic/server.xml [hunk 5]
```diff
@@ -12,7 +12,7 @@
 		ldapType="IBM Tivoli Directory Server"
 		searchTimeout="8m">
 		<failoverServers name="failoverLdapServers">
-			<server host="${ldap.server.4.name}" port="${ldap.server.4.port}"/>
+			<server host="${ldap.server.5.name}" port="${ldap.server.5.port}"/>
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.core_fat/publish/servers/com.ibm.ws.security.wim.core.fat.dynamic/server.xml]: This belongs in PR #5429.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/vmm.apis.tds.ldap/server.xml [hunk 6]
```diff
@@ -29,7 +29,7 @@
       </attributeConfiguration>
       
       <failoverServers name="failoverLdapServers">
-      		<server host="${ldap.server.4.name}" port="${ldap.server.4.port}"/>
+      		<server host="${ldap.server.1.name}" port="${ldap.server.1.port}"/>
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/vmm.apis.tds.ldap/server.xml]: This belongs in PR #5429.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.tworealms/server.xml [hunk 7]
```diff
@@ -56,7 +56,7 @@
         <searchResultsCache size="2000" timeout="600ms" enabled="true" resultsSizeLimit="1000"/>
       </ldapCache>
       <failoverServers name="failoverLdapServers">
-      		<server host="${ldap.server.4.name}" port="${ldap.server.4.port}"/>
+      		<server host="${ldap.server.1.name}" port="${ldap.server.1.port}"/>
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.tworealms/server.xml]: This belongs in PR #5429.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.tds/server.xml [hunk 8]
```diff
@@ -57,7 +57,6 @@
         <searchResultsCache size="2000" timeout="600ms" enabled="true" resultsSizeLimit="1000"/>
       </ldapCache>
       <failoverServers name="failoverLdapServers">
-      		<server host="${ldap.server.4.name}" port="${ldap.server.4.port}"/>
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.tds/server.xml]: This belongs in PR #5429.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.tds.sslref/server.xml [hunk 9]
```diff
@@ -67,6 +67,7 @@
       </ldapCache>
       <failoverServers name="failoverLdapServers">
 		   <server host="${ldap.server.7.name}" port="${ldap.server.7.ssl.port}"/>
+		   <server host="${ldap.server.8.name}" port="${ldap.server.8.ssl.port}"/>
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.tds.sslref/server.xml]: This belongs in PR #5429.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.tds.sslref.badssl/server.xml [hunk 10]
```diff
@@ -68,6 +68,8 @@
       </ldapCache>
       <failoverServers name="failoverLdapServers">
 		   <server host="${ldap.server.7.name}" port="${ldap.server.7.ssl.port}"/>
+		   <server host="${ldap.server.8.name}" port="${ldap.server.8.ssl.port}"/>
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.tds.sslref.badssl/server.xml]: This belongs in PR #5429.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.tds.ssl/server.xml [hunk 11]
```diff
@@ -67,6 +67,7 @@
       </ldapCache>
       <failoverServers name="failoverLdapServers">
 		   <server host="${ldap.server.7.name}" port="${ldap.server.7.ssl.port}"/>
+		   <server host="${ldap.server.8.name}" port="${ldap.server.8.ssl.port}"/>
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.tds.ssl/server.xml]: This belongs in PR #5429.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.tds.nested/server.xml [hunk 12]
```diff
@@ -28,8 +28,8 @@
 			groupIdMap="*:cn"
 			groupMemberIdMap="ibm-allGroups:member;ibm-allGroups:uniqueMember;groupOfNames:member;groupOfUniqueNames:uniqueMember" />
 		<failoverServers name="failoverLdapServers">
-      		<server host="${ldap.server.1.name}" port="${ldap.server.1.port}"/>
-			<server host="${ldap.server.5.name}" port="${ldap.server.5.port}"/>
+      		<server host="${ldap.server.7.name}" port="${ldap.server.7.port}"/>
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.tds.nested/server.xml]: This belongs in PR #5429.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.tds.fedcertlogin/server.xml [hunk 13]
```diff
@@ -35,7 +35,7 @@
 		searchTimeout="8m"
 		loginProperty="cn">
 		<failoverServers name="failoverLdapServers">
-		   <server host="${ldap.server.4.name}" port="${ldap.server.4.port}"/>
+		   <server host="${ldap.server.1.name}" port="${ldap.server.1.port}"/>
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.tds.fedcertlogin/server.xml]: This belongs in PR #5429.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.tds.emptyInput/server.xml [hunk 14]
```diff
@@ -23,7 +23,7 @@
 		ldapType="IBM Tivoli Directory Server"
 		searchTimeout="8m">
 		<failoverServers name="failoverLdapServers">
-      		<server host="${ldap.server.4.name}" port="${ldap.server.4.port}"/>
+      		<server host="${ldap.server.1.name}" port="${ldap.server.1.port}"/>
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.tds.emptyInput/server.xml]: This belongs in PR #5429.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.tds.certlogin_SBInFilter/server.xml [hunk 15]
```diff
@@ -35,7 +35,7 @@
 		searchTimeout="8m"
 		loginProperty="cn">
 		<failoverServers name="failoverLdapServers">
-		   <server host="${ldap.server.4.name}" port="${ldap.server.4.port}"/>
+		   <server host="${ldap.server.1.name}" port="${ldap.server.1.port}"/>
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.tds.certlogin_SBInFilter/server.xml]: This belongs in PR #5429.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.tds.certlogin/server.xml [hunk 16]
```diff
@@ -35,7 +35,7 @@
 		searchTimeout="8m"
 		loginProperty="cn">
 		<failoverServers name="failoverLdapServers">
-		   <server host="${ldap.server.4.name}" port="${ldap.server.4.port}"/>
+		   <server host="${ldap.server.1.name}" port="${ldap.server.1.port}"/>
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.tds.certlogin/server.xml]: This belongs in PR #5429.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.tds.attrMappingVar2/server.xml [hunk 17]
```diff
@@ -56,7 +56,7 @@
         <searchResultsCache size="2000" timeout="600ms" enabled="true" resultsSizeLimit="1000"/>
       </ldapCache>
       <failoverServers name="failoverLdapServers">
-      		<server host="${ldap.server.4.name}" port="${ldap.server.4.port}"/>
+      		<server host="${ldap.server.1.name}" port="${ldap.server.1.port}"/>
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.tds.attrMappingVar2/server.xml]: This belongs in PR #5429.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.tds.attrMappingVar1/server.xml [hunk 18]
```diff
@@ -56,7 +56,6 @@
         <searchResultsCache size="2000" timeout="600ms" enabled="true" resultsSizeLimit="1000"/>
       </ldapCache>
       <failoverServers name="failoverLdapServers">
-      		<server host="${ldap.server.4.name}" port="${ldap.server.4.port}"/>
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.tds.attrMappingVar1/server.xml]: This belongs in PR #5429.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.searchbase/server.xml [hunk 19]
```diff
@@ -61,7 +61,7 @@
       </ldapCache>
       <failoverServers name="failoverLdapServers">
       	<server host="${ldap.server.1.name}" port="${ldap.server.1.port}"/>
-		<server host="${ldap.server.4.name}" port="${ldap.server.4.port}"/>
+		<server host="${ldap.server.5.name}" port="${ldap.server.5.port}"/>
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.searchbase/server.xml]: This belongs in PR #5429.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.multipleldaps/server.xml [hunk 20]
```diff
@@ -42,7 +42,6 @@
 		ldapType="IBM Tivoli Directory Server"
 		searchTimeout="8m">
 		<failoverServers name="failoverLdapServers">
-      		<server host="${ldap.server.4.name}" port="${ldap.server.4.port}"/>
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.multipleldaps/server.xml]: This belongs in PR #5429.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.federation/server.xml [hunk 21]
```diff
@@ -56,7 +56,6 @@
         <searchResultsCache size="2000" timeout="600ms" enabled="true" resultsSizeLimit="1000"/>
       </ldapCache>
       <failoverServers name="failoverLdapServers">
-      		<server host="${ldap.server.4.name}" port="${ldap.server.4.port}"/>
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.federation/server.xml]: This belongs in PR #5429.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.registry.ldap.fat.performance/server.xml [hunk 22]
```diff
@@ -29,7 +29,6 @@
 			groupMemberIdMap="ibm-allGroups:member;ibm-allGroups:uniqueMember;groupOfNames:member;groupOfUniqueNames:uniqueMember" />
 		<failoverServers name="failoverLdapServers">
 		   <server host="${ldap.server.5.name}" port="${ldap.server.5.port}"/>
-           <server host="${ldap.server.4.name}" port="${ldap.server.4.port}"/>
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.registry.ldap.fat.performance/server.xml]: This belongs in PR #5429.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.registry.ldap.fat.ldapname/server.xml [hunk 23]
```diff
@@ -30,7 +30,6 @@
 			groupMemberIdMap="ibm-allGroups:member;ibm-allGroups:uniqueMember;groupOfNames:member;groupOfUniqueNames:uniqueMember" />
 		<failoverServers name="failoverLdapServers">
 		   <server host="${ldap.server.5.name}" port="${ldap.server.5.port}"/>
-           <server host="${ldap.server.4.name}" port="${ldap.server.4.port}"/>
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.registry.ldap.fat.ldapname/server.xml]: This belongs in PR #5429.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.registry.ldap.fat.ids/server.xml [hunk 24]
```diff
@@ -30,8 +30,8 @@
 			groupIdMap="*:cn"
 			groupMemberIdMap="groupOfNames:member;groupOfUniqueNames:uniqueMember" />
 		<failoverServers name="failoverLdapServers">
-		   <server host="${ldap.server.5.name}" port="${ldap.server.5.port}"/>
-           <server host="${ldap.server.1.name}" port="${ldap.server.1.port}"/>
+		   <server host="${ldap.server.7.name}" port="${ldap.server.7.port}"/>
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.registry.ldap.fat.ids/server.xml]: This belongs in PR #5429.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.registry.ldap.fat.ids.ssl/server.xml [hunk 25]
```diff
@@ -28,6 +28,7 @@
 		idsFilters="ibm_dir_server">
 		<failoverServers name="failoverLdapServers">
 		   <server host="${ldap.server.7.name}" port="${ldap.server.7.ssl.port}"/>
+		   <server host="${ldap.server.8.name}" port="${ldap.server.8.ssl.port}"/>
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.registry.ldap.fat.ids.ssl/server.xml]: This belongs in PR #5429.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.registry.ldap.fat.ids.ssl.trustonly/server.xml [hunk 26]
```diff
@@ -29,6 +29,7 @@
 		idsFilters="ibm_dir_server">
 		<failoverServers name="failoverLdapServers">
 		   <server host="${ldap.server.7.name}" port="${ldap.server.7.ssl.port}"/>
+		   <server host="${ldap.server.8.name}" port="${ldap.server.8.ssl.port}"/>
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.registry.ldap.fat.ids.ssl.trustonly/server.xml]: This belongs in PR #5429.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.registry.ldap.fat.ids.reuseConnection/server.xml [hunk 27]
```diff
@@ -30,7 +30,6 @@
 			groupMemberIdMap="ibm-allGroups:member;ibm-allGroups:uniqueMember;groupOfNames:member;groupOfUniqueNames:uniqueMember" />
 		<failoverServers name="failoverLdapServers">
 		   <server host="${ldap.server.5.name}" port="${ldap.server.5.port}"/>
-           <server host="${ldap.server.4.name}" port="${ldap.server.4.port}"/>
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.registry.ldap.fat.ids.reuseConnection/server.xml]: This belongs in PR #5429.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.registry.ldap.fat.ids.nofilters/server.xml [hunk 28]
```diff
@@ -23,7 +23,6 @@
 		searchTimeout="8m">
 		<failoverServers name="failoverLdapServers">
 		   <server host="${ldap.server.5.name}" port="${ldap.server.5.port}"/>
-           <server host="${ldap.server.4.name}" port="${ldap.server.4.port}"/>
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.registry.ldap.fat.ids.nofilters/server.xml]: This belongs in PR #5429.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.registry.ldap.fat.ids.failover/server.xml [hunk 29]
```diff
@@ -22,8 +22,8 @@
 		searchTimeout="8m"
 		idsFilters="ibm_dir_server">
 		<failoverServers name="failoverLdapServers">
-           <server host="${ldap.server.1.name}" port="${ldap.server.1.port}"/>
-           <server host="${ldap.server.5.name}" port="${ldap.server.5.port}"/>
+           <server host="${ldap.server.7.name}" port="${ldap.server.7.port}"/>
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.registry.ldap.fat.ids.failover/server.xml]: This belongs in PR #5429.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.registry.ldap.fat.ids.extremeFailover/server.xml [hunk 30]
```diff
@@ -27,7 +27,6 @@
 			<server host="${ldap.server.1.port}" port="379"/>
 		</failoverServers>
 		<failoverServers name="failoverLdapServersGroup2">
-			<server host="${ldap.server.4.name}" port="${ldap.server.4.port}"/>
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.registry.ldap.fat.ids.extremeFailover/server.xml]: This belongs in PR #5429.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.registry.ldap.fat.custom/server.xml [hunk 31]
```diff
@@ -23,8 +23,8 @@
 		recursiveSearch="true"
 		customFilters="mycustom">
 		<failoverServers name="failoverLdapServers">
-		   <server host="${ldap.server.5.name}" port="${ldap.server.5.port}"/>
-           <server host="${ldap.server.1.name}" port="${ldap.server.1.port}"/>
+		   <server host="${ldap.server.7.name}" port="${ldap.server.7.port}"/>
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.registry.ldap.fat.custom/server.xml]: This belongs in PR #5429.

File: dev/com.ibm.ws.security.wim.adapter.ldap/test/com/ibm/ws/security/wim/adapter/ldap/context/ContextManagerTest.java [hunk 32]
```diff
@@ -781,25 +781,40 @@ public void connectTimeout() throws Exception {
     public void readTimeout() throws Exception {
 
         /*
-         * Configure the context manager with a 100 ms read timeout.
+         * Configure the context manager with a ms read timeout.
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap/test/com/ibm/ws/security/wim/adapter/ldap/context/ContextManagerTest.java]:
```suggestion
         * Configure the context manager with the default read timeout (60000).
```

File: dev/fattest.simplicity/src/com/ibm/websphere/simplicity/config/wim/LdapRegistry.java [hunk 33]
```diff
@@ -629,6 +653,14 @@ public void setSslRef(String sslRef) {
         this.sslRef = sslRef;
     }
 
+    /**
```
Comment (jvanhill) [inline dev/fattest.simplicity/src/com/ibm/websphere/simplicity/config/wim/LdapRegistry.java]: Looks like this is in PR #5429 as well.

File: dev/fattest.simplicity/src/com/ibm/websphere/simplicity/config/wim/LdapRegistry.java [hunk 34]
```diff
@@ -743,6 +778,9 @@ public String toString() {
         if (sslRef != null) {
             sb.append("sslRef=\"").append(sslRef).append("\" ");;
         }
+        if (timestampFormat != null) {
```
Comment (jvanhill) [inline dev/fattest.simplicity/src/com/ibm/websphere/simplicity/config/wim/LdapRegistry.java]: Looks like this is in PR #5429 as well.
